app/templates/components/logo-banner.hbs

```diff
@@ -5,8 +5,12 @@
             <div class="contributor jockey_club"> {{ t "banner.funded_by" }} </div>
         </div>
         <div class="contributors logos">
-            <div class="contributor logo crossroads"></div>
-            <div class="contributor logo jockey_club"></div>
+            <div class="contributor crossroads">
```

Comment:
indentation 😄

Reply:
Every time I configure my editor, it resets itself  😞